libraries/AC_AttitudeControl/AC_AttitudeControl.cpp

```diff
@@ -204,6 +225,21 @@ void AC_AttitudeControl::reset_rate_controller_I_terms_smoothly()
     get_rate_yaw_pid().relax_integrator(0.0, _dt, AC_ATTITUDE_RATE_RELAX_TC);
 }
 
+// Reduce attitude control gains while landed to stop ground resonance
+void AC_AttitudeControl::landed_gain_reduction(bool landed)
+{
+    // use 2.0 x tc to match the response time to 86% commanded
+    const float spool_step = _dt / (2.0 * _input_tc);
```

Comment:
I wonder if we should catch the possible divide by zero here

Reply:
I am not sure if I like what I did here.